ext/src/Cassandra/Numeric.c

```diff
@@ -0,0 +1,35 @@
+#include <php.h>
+#include "php_cassandra.h"
+
+zend_class_entry* cassandra_numeric_ce = NULL;
+
+ZEND_BEGIN_ARG_INFO_EX(arginfo_none, 0, ZEND_RETURN_VALUE, 0)
+ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_INFO_EX(arginfo_num, 0, ZEND_RETURN_VALUE, 1)
+  ZEND_ARG_INFO(0, num)
+ZEND_END_ARG_INFO()
+
+static zend_function_entry cassandra_numeric_methods[] = {
+  PHP_ABSTRACT_ME(Numeric, add, arginfo_num)
+  PHP_ABSTRACT_ME(Numeric, sub, arginfo_num)
+  PHP_ABSTRACT_ME(Numeric, mul, arginfo_num)
+  PHP_ABSTRACT_ME(Numeric, div, arginfo_num)
+  PHP_ABSTRACT_ME(Numeric, mod, arginfo_num)
+  PHP_ABSTRACT_ME(Numeric, abs, arginfo_none)
+  PHP_ABSTRACT_ME(Numeric, neg, arginfo_none)
+  PHP_ABSTRACT_ME(Numeric, sqrt, arginfo_none)
+  PHP_ABSTRACT_ME(Numeric, toInt, arginfo_none)
+  PHP_ABSTRACT_ME(Numeric, toDouble, arginfo_none)
```

Comment:
`toFloat()` is pretty confusing with there being a `Cassandra\Float` type. There are also "double" aliases for most "float" related things in PHP (unlike for "int"). It came down to time for `toVarint` etc. I did implement them so I didn't add them.